.clang-format

```diff
@@ -5,6 +5,42 @@ BasedOnStyle: LLVM
 Language: Cpp
 Standard: c++20
 
+AccessModifierOffset: -4
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: false
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: false
+BinPackParameters: false
+BraceWrapping:
+  AfterCaseLabel: true
+  AfterClass: true
+  AfterControlStatement: Always
+  AfterEnum: true
+  AfterExternBlock: true
+  AfterFunction: true
+  AfterNamespace: true
+  AfterObjCDeclaration: true
+  AfterStruct: true
+  AfterUnion: true
+  BeforeCatch: true
+  BeforeElse: true
+BreakBeforeBraces: Custom
+ColumnLimit: 100
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+Cpp11BracedListStyle: true
+DerivePointerAlignment: false
+IndentExternBlock: NoIndent
+IndentWidth: 4
+InsertNewlineAtEOF: true
+PenaltyReturnTypeOnItsOwnLine: 120
+PointerAlignment: Left
```

Comment:
We can move these to a section where no language is set, like in libcxx's .clang-format file:

https://github.com/llvm/llvm-project/blob/main/libcxx/.clang-format

This way all langs would inherit from it. Then, we can define language-specific parts as needed.